hadoop-tools/hadoop-aws/src/main/java/org/apache/hadoop/fs/common/CachingBlockManager.java

```diff
@@ -318,6 +328,7 @@ private void readBlock(BufferData data, boolean isPrefetch, BufferData.State...
         }
 
         if (isPrefetch) {
+          prefetchingStatistics.prefetchOperationStarted();
```

Comment:
have this return a DurationTracker whose close() will update the statistic. on an an exception, call its failed() method to update the .failure keys intead